homatransport/src/common/AppmasterBase.ned

```diff
@@ -28,7 +28,7 @@ import homatransport.transport.ITransportScheme;
 // unnecessary parts are trimmed off and TransportScheme has been added.
 //
```

Comment:
Where does appmaster run? what is it's job? How is it supposed to be used?
Please check the styleguide https://google.github.io/styleguide/cppguide.html#Comments for how to write useful comments. Generally whatever is not obvious from code, should be in comments.